.github/workflows/on-release.yml

```diff
@@ -59,10 +58,17 @@ jobs:
           aws s3 cp '${{ env.S3BUCKET_URL }}/${{ env.S3BUCKET_FILE }}' \
              'aleph-node-${{ steps.get-ref-properties.outputs.sha }}-linux-amd64.tar.gz'
 
+      - name: Generate release artifacts checksum (SHA256)
+        uses: jmgilman/actions-generate-checksum@v1
+        with:
+          patterns: |
+            *.tar.gz
+
       - name: Add runtime and binary to the release
         uses: softprops/action-gh-release@v1
         with:
           files: |
+            checksum.txt
```

Comment:
Shouldn't we call the file something else, like `aleph-node-${{ steps.get-ref-properties.outputs.sha }}-linux-amd64.tar.gz.$ALGO` where `$ALGO` is the algorithm like `sha256` or whatever...

Reply:
Also, shouldn't we create a check sum the runtime as well?

Reply:
Rework pushed